ApplyToBecomeInternal/ApplyToBecomeInternal/Models/AcademyConversionProjectPostModel.cs

```diff
@@ -148,7 +148,14 @@ public class AcademyConversionProjectPostModel
 		[DisplayFormat(ConvertEmptyStringToNull = false)]
 		public string SchoolPupilForecastsAdditionalInformation { get; set; }
 
-		//key stage performance tables
+		// key stage performance tables
 		public bool? KeyStagePerformanceTablesAdditionalInformation { get; set; }
+
+		// project note
+		[BindProperty(Name = "project-note-subject")]
+		public string ProjectNoteSubject { get; set; }
+
+		[BindProperty(Name = "project-note-body")]
+		public string ProjectNoteBody { get; set; }

```

Comment:
This should probably be its own model as it doesn't really relate to the same entity